### PR TITLE
Include scheduled slots in pools view

### LINKED ISSUE #25976: Include "Scheduled slots" column in Pools view
### Description

It would be nice to have a "Scheduled slots" column to see how many slots want to enter each pool. Currently we are only displaying the running and queued slots.

### Use case/motivation

_No response_

### Related issues

_No response_

### Are you willing to submit a PR?

- [X] Yes I am willing to submit a PR!

### Code of Conduct

- [X] I agree to follow this project's [Code of Conduct](https://github.com/apache/airflow/blob/main/CODE_OF_CONDUCT.md)

### PR DESCRIPTION
**Pull request by csp33**
closes: #25976

### CODE REVIEW

**Comment by BasPH** [comment]
Could you show a screenshot?

**Comment by csp98** [comment]
Sure!
image

**Review by uranusjr — APPROVED** [review]

File: airflow/api_connexion/schemas/pool_schema.py [hunk 1]
```diff
@@ -54,6 +55,11 @@ def get_queued_slots(obj: Pool) -> int:
         """Returns the queued slots of the pool."""
         return obj.queued_slots()
 
+    @staticmethod
+    def get_scheduled_slots(obj: Pool) -> int:
+        """Returns the queued slots of the pool."""
```
Comment (BasPH) [inline airflow/api_connexion/schemas/pool_schema.py]:
```suggestion
        """Returns the scheduled slots of the pool."""
```

**Review by eladkal — APPROVED** [review]

**Comment by csp98** [comment]
Ready to merge! @eladkal @uranusjr

**Pull request merged**